### PR TITLE
Finish task static pages

### PR DESCRIPTION
**Pull request by tqtrung09**

### CODE REVIEW

File: config/initializers/config.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Config.setup do |config|
+  config.const_name = "Settings"
+end
```
Comment (lyxuansang91) [inline config/initializers/config.rb]: end file

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+
+  def full_title page_title = ""
+    base_title = t "views.helper.base_title"
+    if page_title.empty?
```
Comment (hellovietnam93) [inline app/helpers/application_helper.rb]: page_title.empty? ? base_title : "#{page_title} | #{base_title}"

File: config/locales/en.yml [hunk 3]
```diff
@@ -20,4 +20,14 @@
...
+  help: "Help"
+  tms49: "TMS-49"
+  views:
+    footer:
+      me: "Tran.Quang.Trung@framgia.com"
+    helper:
+      base_title: "Ruby on Rails Tutorial TMS-49"
+    header:
+      home: "Home"
+
```
Comment (hellovietnam93) [inline config/locales/en.yml]: remove this line

**Comment by hellovietnam93** [comment]
OK

**Pull request merged**